src/index.tsx

```diff
@@ -71,6 +76,7 @@ function Updaters() {
       <CancelledOrdersUpdater />
       <FeesUpdater />
       <UnfillableOrdersUpdater />
+      <APIOrdersUpdater />
```

Comment:
Because there are 3 other OrdersUpdater already:
![screenshot_2021-10-05_11-01-56](https://user-images.githubusercontent.com/43217/136077982-7f63eaae-4f2b-400d-85bd-f08894a6895f.png)

Having one that's just `OrdersUpdaters` would be ambiguous IMO

Reply:
🙈